pkgs/by-name/sc/scraparr/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "scraparr";
+  version = "2.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "thecfu";
+    repo = "scraparr";
+    tag = "v${version}";
+    hash = "sha256-hU1/0zv4dJ8kHo5MdaJqFF04lpnjQbfxvWQ4IaodIzo=";
+  };
+
+  build-system = with python3.pkgs; [ setuptools ];
+
+  dependencies = with python3.pkgs; ([
+    prometheus-client
+    requests
+    werkzeug
+    python-dateutil
+    pyyaml
+    python-dotenv
+  ]);
+
+  pythonImportsCheck = [ "scraparr" ];
+
+  postPatch = ''
+    substituteInPlace src/scraparr/scraparr.py \
+      --replace-fail 'CONFIG_FILE_LOCATION = "/scraparr/config/config.yaml"' 'CONFIG_FILE_LOCATION = "/etc/scraparr/config.yaml"'
+  '';
+
+  postInstall = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib/${pname}
+
+    cp -r ./src/scraparr $out/lib/${pname}/
+
+    cat > $out/bin/scraparr << EOF
+    #!${python3.interpreter}
+    import sys
+    sys.path.insert(0, "$out/lib/${pname}")
+    import scraparr.scraparr
+    EOF
+
+    chmod +x $out/bin/scraparr
+
+    wrapProgram $out/bin/scraparr \
+      --prefix PATH : "${lib.makeBinPath dependencies}" \
+      --prefix PYTHONPATH : "$out/${python3.sitePackages}"
```

Comment:
Using the patch from the PR created by @mweinelt would be the recommended way to do this, but for the record, here's some info on packaging python packages:

You don't need to do `cp -r ./src/scraparr $out/lib/${pname}/` because the `scraparr` modules is already installed into `$out/lib/python3.13/site-packages/`.

Also, you don't need to add to `PATH` and `PYTHONPATH` because the python scripts in `$out/bin` are automatically wrapped in a way where `PYTHONPATH` is set up correctly (the PATH is also populated automatically from dependencies, but I don't know if you actually need this here).

Reply:
Gotcha. Thanks!